### DISCUSSION TITLE
[Feature] Ability to select all libraries on your devices

### DISCUSSION THREAD

**Discussion by usshauler** [immich-app/immich#11339] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add the ability to select all libraries within your iOS or Android device settings versus having to go 1 by 1 and manually check box each library that you want as a backup.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by alextran1502** [comment]
On iOS or Android, the album that contains everything is called `Recent` or `Recents`
